docs/whats-new.md

```diff
@@ -1,36 +1,62 @@
 
-* Support for `scratch`
+- Import directives can include destination paths. This feature is useful when images are built from a scratch layer before any shell has been invoked. Prior to this, a `run:` section was required to invoke a shell and to explicitly copy files to be imported into the image. For example, you can now write a directive such as the following, with no `run:` section:
 
-Prior to v0.40.1, `stacker` did not support empty root filesystems to be used a
-base container image. The support has now been [added](reference/stacker_file.md#from) which can be used to host
-statically built binaries.
+        test:
+          from:
+            type: scratch
+          imports:
+            - path: test_file
+              dest: /files/
+            - path: test_file2
+              dest: /file2
 
-* Support for `import`ing content into container image
+### Generate SBOMs during the build
+
+- Changes added in [OCI Distribution Spec v1.1.0-rc3](https://github.com/opencontainers/distribution-spec/releases/tag/v1.1.0-rc3) and [OCI Image Spec v1.1.0-rc4](https://github.com/opencontainers/image-spec/releases/tag/v1.1.0-rc4) (summarized [here](https://opencontainers.org/posts/blog/2023-07-07-summary-of-upcoming-changes-in-oci-image-and-distribution-specs-v-1-1/)) allow arbitrary artifact types and references. These changes support software supply chain use cases such as SBOMs.
+
+- For a demonstration of an OCI artifacts workflow that generates an SBOM, see [Software Provenance Workflow Using OCI Artifacts](user_guide/generate_sbom.md).
+
+### Report kernel version and fs type
+
+- The [`stacker check`](reference/stacker_cli.md#stacker-check) command now reports this information.
+
+### Build improvements
+
+***
+
+## [v0.40.1](https://github.com/project-stacker/stacker/releases/tag/v0.40.1)
 
-Prior to v0.40.1, copying content into a layer permanently involved bind
-mounting a shell such as busybox and invoking appropriate commands using the
-`run` directive. Now `import` directive [allows](reference/stacker_file.md#import-dest) for the `dest` option to achieve
-the same.
+### Support for `scratch`
 
-* For the `build` command, can specify substitute key-value pairs in a file instead of the commandline.
+- Prior to v0.40.1, `stacker` did not support empty root filesystems to be used a base container image. The support has now been [added](reference/stacker_file.md#from) which can be used to host statically built binaries.
 
-```
-$ stacker build --help
-NAME:
-   stacker build - builds a new OCI image from a stacker yaml file
+### Support for `import`ing content into container image
 
-USAGE:
-   stacker build [command options] [arguments...]
+- Prior to v0.40.1, copying content into an image permanently involved bind mounting a shell such as busybox and invoking appropriate commands using the `run` directive. Now `import` directive [allows](reference/stacker_file.md#import-dest) for the `dest` option to achieve the same.
```

Comment:
I missed this earlier, but this paragraph is only relevant in the context of scratch images. you could always copy into an image without bind mounting a shell if your base image had a shell already.

maybe this should read "prior to v0.40.1, copying content into a `scratch` image ..."